packages/docusaurus/src/commands/start.ts

```diff
@@ -197,6 +197,12 @@ export default async function start(
         ...{pollingOptions},
       },
     })),
+    ...(httpsConfig && {
+      server: {
+        type: 'https',
+        options: httpsConfig as ServerOptions,
```

Comment:
Our `getHttpsConfig` returns `true` if the env says `HTTPS=true` but doesn't provide the key and cert. Is that actually a legitimate situation to be in? Should we actually return `false | ServerOptions` instead, and throw an error if there's no key?

Reply:
Yes it is valid, in this case a self-signed certificate will be used, just fixed it.